src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/XamarinAndroidApplicationProject.cs

```diff
@@ -66,8 +67,8 @@ public XamarinAndroidApplicationProject (string debugConfigurationName = "Debug"
 				SetProperty (ReleaseProperties, "AndroidLinkMode", "SdkOnly");
 				SetProperty (DebugProperties, KnownProperties.EmbedAssembliesIntoApk, "False", "'$(EmbedAssembliesIntoApk)' == ''");
 				SetProperty (ReleaseProperties, KnownProperties.EmbedAssembliesIntoApk, "True", "'$(EmbedAssembliesIntoApk)' == ''");
+				MinSdkVersion = "19";
```

Comment:
This feels wrong; shouldn't it be `21`?

Reply:
Why move this into the constructor and not leave it in the property declaration?

Reply:
For that matter, couldn't this entire `else` block just be removed now?  It's for Classic, isn't it?

Reply:
This change ensure that we will no longer explicitly write `android:minSdkVersion="19"` to every test projects AndroidManifest.xml. I could just remove this now however, as it only applies to classic tests. I don't want to expand scope too much by stripping out pivots for classic tests in this PR though.

Reply:
This setting is for Classic, not .NET Android.  We should remove in a future PR.